docs/getting-started/generated-app.md

```diff
@@ -36,24 +40,41 @@ The generated service is built with the following proven open-source technologie
 - [Jest](https://jestjs.io/) - delightful JavaScript Testing Framework with a focus on simplicity.
 - [Docker](https://www.docker.com/) - an open platform for developing, shipping, and running applications.
 
+#### .NET
```

Comment:
@mulygottlieb - Can you please review and update it if needed?

Reply:
@dericksozo @PazYanoverr 
Let's remove IdentityServer and XUnit from the list for now. 

I am also not sure if we want to add SQL Server since we are in general DB-agnostic and perhaps we don't want to limit ourselves in the docs. The same is true BTW for specifying PostgreSQL in the Node.js section - we support many databases in Node.js and will support many DBs on .NET too eventually - so perhaps it's better not to mention a single DB as that sounds too restrictive IMO. @PazYanoverr  your call.

Reply:
@mulygottlieb - I agree, although in node.js we mention not only postgres but also mongo and mysql.
@dericksozo - Let's write something more "vague". In node.js- we need to say that we support many databases- like postgres, mongo, mysql, mssql etc. In .NET- we need to say the same while giving the mssql as an example.
@mulygottlieb - Are we going to support mssql as the first database?